server/app.js

```diff
@@ -68,4 +69,8 @@ process.on("unhandledRejection", (err, promise) => {
   server.close(() => process.exit(1));
 });
 
+app.listen(port, () => {
+  console.log(`Example app listening at http://localhost:${port}`)
+})
```

Comment:
We already have a startup script - it's called `bin/www` you can run the server by doing `npm run dev`